src/components/questions/OpenTextArea.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { StyleSheet, TextInput, TextInputProps } from 'react-native';
+import FONTS from '../../assets/fonts';
+
+const OpenTextArea = (props: TextInputProps) => {
+  return (
+    <TextInput
+      {...props}
+      multiline
+      numberOfLines={5}
+      textAlignVertical="top"
+      style={styles.openTextArea}
+    />
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Misschien een idee om er een max height aan toe te voegen, stel iemand typt een heel boekwerk, krijg je een heel groot input veld ;)

Reply:
Oh dit wordt geregeld dankzij dit `numberOfLines={5}`. Zorgt er automatisch voor dat je kan scrollen in het vakje na 5 regels tekst

Reply:
Ik gok dat t niet werkt op IOS, want ik kreeg echt een heel groot veld, was misschien wel 10 enters of meer